server/index.js

```diff
@@ -35,24 +37,24 @@ const checkout = new CheckoutAPI(client);
 app.post("/api/sessions", async (req, res) => {
   const localhost = req.get('host');
   try {
-    // unique ref for the transaction
+    // Unique ref for the transaction
     const orderRef = uuid();
 
     const protocol = req.socket.encrypted? 'https' : 'http';
     const host = req.get('host');
 
     // Ideally the data passed here should be computed based on business logic
     const response = await checkout.sessions({
-      amount: { currency: "EUR", value: 10000 }, // value is 100€ in minor units
+      amount: { currency: "EUR", value: 10000 }, // Value is 100€ in minor units
       countryCode: "NL",
-      merchantAccount: process.env.ADYEN_MERCHANT_ACCOUNT, // required
-      reference: orderRef, // required: your Payment Reference
+      merchantAccount: process.env.ADYEN_MERCHANT_ACCOUNT, // Required: your merchant account
+      reference: orderRef, // Required: your Payment Reference
       // set lineItems required for some payment methods (ie Klarna)
       lineItems: [
         {quantity: 1, amountIncludingTax: 5000 , description: "Sunglasses"},
         {quantity: 1, amountIncludingTax: 5000 , description: "Headphones"}
       ] ,
-      returnUrl: `${protocol}://${host}/api/handleShopperRedirect?orderRef=${orderRef}` // set redirect URL required for some payment methods
+      returnUrl: `${protocol}://${host}/api/handleShopperRedirect?orderRef=${orderRef}` // Set redirect URL required for some payment methods
```

Comment:
Like all other comments we can also indicate here that `returnUrl` is a required param